workflows/dependabot.yml

```diff
@@ -0,0 +1,46 @@
+name: dependabot
```

Comment:
Was this file supposed to be created in `$PROJECT_ROOT/.github/workflows/` dir than in `$PROJECT_ROOT/workflows/` dir?

Reply:
+1